### PR TITLE
changes for concourse ci integration

### PR DESCRIPTION
**Pull request by alexkonkin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,50 +12,53 @@
...
 # Docker build config variables
-CREDENTIALS_PATH ?= /cft/workdir/credentials.json
-DOCKER_ORG := gcr.io/cloud-foundation-cicd
-DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 0.11.10_216.0.0_1.19.1_0.1.10
-DOCKER_REPO_BASE_KITCHEN_TERRAFORM := ${DOCKER_ORG}/cft/kitchen-terraform:${DOCKER_TAG_BASE_KITCHEN_TERRAFORM}
-DOCKER_TAG_KITCHEN_TERRAFORM ?= ${DOCKER_TAG_BASE_KITCHEN_TERRAFORM}
-DOCKER_IMAGE_KITCHEN_TERRAFORM := cft/kitchen_terraform
+CREDENTIALS_PATH 			?= /cft/workdir/credentials.json
+DOCKER_ORG 				:= gcr.io/cloud-foundation-cicd
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	?= 1.0.1
```
Comment (aaron-lane) [inline Makefile]: Let's us the latest version to match the CI pipeline:

```suggestion
DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	?= 1.2.0
```

Comment (alexkonkin) [reply Makefile]: Aaron, thank you for your review. Done.

File: test/ci_integration.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Always clean up.
+DELETE_AT_EXIT="$(mktemp -d)"
+finish() {
+  echo 'BEGIN: finish() trap handler' >&2
+  bundler exec kitchen destroy "$SUITE"
```
Comment (aaron-lane) [inline test/ci_integration.sh]:
```suggestion
  kitchen destroy "$SUITE"
```

Comment (alexkonkin) [reply test/ci_integration.sh]: Thank you for your review. The odd part of the command has been removed.

**Pull request merged**